public/stylesheets/style.css

```diff
@@ -30,3 +30,19 @@ a, li a {
   height: 390px;
   overflow-y: scroll;
 }
+
+.right {
+    float: right;
+}
+
+.positions {
+  font-size: 70%;
+}
+
+.position {
+  padding: 2px;
+}
+
+.taken {
+  text-decoration: line-through;
```

Comment:
I was just looking at the screenshot again. I should have added color: green in .position, and color: red for .taken. We can add that later.